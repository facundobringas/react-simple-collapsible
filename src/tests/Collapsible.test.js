import {Collapsible} from '../index';
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

test('Search renders correctly', () => {
    const component = renderer.create(<Collapsible />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

it('renders without crashing', () => {
    shallow(<Collapsible />);
});