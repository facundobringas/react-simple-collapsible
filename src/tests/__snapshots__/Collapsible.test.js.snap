// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<div
  className="collapsible collapsed"
  id="collapsible_0"
  onClick={[Function]}
>
  <div
    className="collapsible-header"
    id="collapsible_header_0"
  >
    
    <img
      className="collapsible-header-arrow"
      id="collapsible_header_arrow_0"
      src="test-file-stub"
    />
  </div>
  <div
    className="collapsible-content"
    id="collapsible_content_0"
  >
    <div
      className="collapsible-content-wrapper"
    />
  </div>
</div>
`;
